packages/config/config/devices/0x0190/adc-t_3000.json

```
@@ -0,0 +1,2839 @@
+{
+	"manufacturer": "Building 36 Technologies",
+	"manufacturerId": "0x0190",
+	"label": "ADC-T 3000",
+	"description": "Alarm.com Smart Thermostat",
+	"devices": [
+		{
+			"productType": "0x0006",
+			"productId": "0x0001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "HVAC System Type",
+			"description": "Configures the type of heating system used.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Normal",
+					"value": 0
+				},
+				{
+					"label": "Heat Pump",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Number of Heat Stages",
+			"description": "Heat Stages 0-3 Default is 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2
+		},
+		{
+			"#": "3",
+			"label": "Number of Cool Stages",
+			"description": "Cool Stages 0-2 Default is 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2
+		},
+		{
+			"#": "4",
+			"label": "Heat Fuel Type",
+			"description": "Choose type of fuel. Reality - whether unit is boiler vs forced air.",
```

The first part should be removed
```suggestion
			"description": "Reality - whether unit is boiler vs forced air.",
```
but even then the description is a bit weird. Maybe these options should be merged with the existing ones, like "Fossil Fuel / Boiler", "Electric / Forced air"?

---

this description is actually copied from the T2000 config, since a good chunk of the configuration overlaps I used it as a base. There is no description in the manual so I'll leave it blank.